extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/operations/KuduWriter.java

```diff
@@ -1,12 +1,10 @@
 package org.locationtech.geowave.datastore.kudu.operations;
 
-import org.apache.kudu.client.Insert;
-import org.apache.kudu.client.KuduException;
-import org.apache.kudu.client.KuduSession;
-import org.apache.kudu.client.PartialRow;
+import org.apache.kudu.client.*;
```

Comment:
I'd rather not have wildcard imports since it can import classes that are not needed